fineract-provider/src/main/java/org/apache/fineract/infrastructure/crypt/domain/EncryptionKeys.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.crypt.domain;
+
+import org.joda.time.LocalDateTime;
+
+/**
+ * @author manoj
+ */
+public final class EncryptionKeys {
```

Comment:
just about the naming - it's confusing (to me) that you name this Keys - in plural. This seems to be A (single, one) key - with it's private and public part. So perhaps better name this a ... PrivateAndPublicKeyWithVersion - kind of thing?

Reply:
Changed it, i was referring the Key Pair as Keys.